script/upload_to_aptly

```diff
@@ -19,22 +19,22 @@ set -exu
 # 12/29/2018    Nightly       0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 
 ### The first argument is the S3 bucket, the second is the GPG key fingerprint
-OPTION=${1:-brave-browser-apt-staging-nightly}
-GPG_KEY_ID=${2:-D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821} # Defaults to New Brave Software signing key
+OPTION=${1:-brave-browser-apt-staging-release-test}
```

Comment:
If we always set this parameter, then I guess we don't really need a default value here and could just error out if any of the parameters are missing.

Reply:
Resolved by removing default values